lib/MadNLPHSL/src/ma86.jl

```diff
@@ -132,7 +132,11 @@ function Ma86Solver(
     csc::SparseMatrixCSC{T,Int32};
     opt=Ma86Options(),logger=MadNLPLogger(),
 ) where T
-    ma86_set_num_threads(opt.ma86_num_threads)
+
+    # Note: the current version of HSL_jll does not support openmp.
+    # this will be reenabled once openmp is supported in HSL_jll.
```

Comment:
`OpenMP` is not supported?!
Did I do a mistake when I compiled HSL_jll.jl?

Reply:
@sshin23 I fixed the issue upstream.
I linked with `-fopenmp` but I didn't compiled with `-fopenmp`.

Reply:
Thanks for the quick action @amontoison! Indeed we used to have `-fopenmp` flag in our previous compilation script 
https://github.com/MadNLP/MadNLP.jl/blob/29e95a5e4f5806778524dc576b8f7223f076bbdf/lib/MadNLPHSL/deps/build.jl#L118
If you want to check it before releasing it, I'd be happy to test it with MadNLPHSL

Reply:
```
x86_64-linux-gnu-libgfortran5-cxx11-libstdcxx30-julia_version+1.9.1
```